app/server/app.js

```diff
@@ -12,4 +13,13 @@ app.get('/', (request, response) => {
   response.sendFile(indexPath);
 });
 
+app.get('/articles', searchArticles, (request, response) => {
+  response.status(200);
+
+  // sends back an array of articles for a single source
+  // not an array of sources
+  const { articles } = request;
+  response.end(JSON.stringify(articles));
```

Comment:
is the `JSON.stringify()` needed? express might do it for us. Or maybe we use `res.json()` instead? I know I have used it before, but not sure how it works. Just some ideas.

Reply:
response.json(articles) or response.end(JSON.stringify(articles)) both work, but response.end(articles) does not. I've changed to response.json(articles), since that's a cleaner naming convention.